### PR TITLE
Mangooio

### PR DESCRIPTION
**Pull request by svenkubiak**
I've added a new framework for benchmark testing: mangoo I/O.

As I am to late for round 11, is there any rough schedule for round 12?

Cheers,
Sven

### CODE REVIEW

File: frameworks/Java/mangooio/src/main/resources/ehcache.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<ehcache xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../config/ehcache.xsd" updateCheck="false">
```
Comment (martin-g) [inline frameworks/Java/mangooio/src/main/resources/ehcache.xml]: Is this used by the test?
AFAIK caching is not allowed.

Comment (svenkubiak) [inline frameworks/Java/mangooio/src/main/resources/ehcache.xml]: No, a Cache is not used at all. But the Framework has an configuration for ehCache which creates an instance of a Cache provider. I could remove the file, but ehCache would still autoconfigure.

**Comment by mcocciaTE** [comment]
I believe you still need to add mangooio to the ./travis.yml if you could add it in alphabetical order that would be awesome.

**Comment by msmith-techempower** [comment]
Travis agrees; LGTM!

**Pull request merged**